app/views/content/ways-to-train.md

```diff
@@ -42,208 +51,150 @@ keywords:
+  <div class="training-option">
+    <dt>School direct<br/>(salaried)</dt>
+    <dd class="badges">
+      <span class="badge purple">Takes 1 year</span>
+      <span class="badge dark-blue">Often a PGCE or PGDE</span>
+    </dd>
+  </div>
 
-You’ll learn on the job while you work towards qualified teacher status (QTS) and a Postgraduate Diploma in Education (PGDE).
+  <div class="training-option">
+    <dt>Postgraduate teaching apprenticeship</dt>
+    <dd class="badges">
+      <span class="badge purple">Takes 1 year</span>
+    </dd>
+  </div>
 
-A PGDE contains up to 120 credits at master’s degree level.
+  <div class="training-option">
+    <dt>Teach First</dt>
+    <dd class="badges">
+      <span class="badge purple">Takes 2 years</span>
+      <span class="badge dark-blue">PGDE</span>
+    </dd>
+  </div>
+</dl>
 
-<p class="call-to-action__action">
-  <a href="https://www.teachfirst.org.uk/">Find a Teach First <span>course</span></a>
-</p>
+$find-postgraduate-training$
 
-$undergraduate$
+## If you do not have a degree
 
- You can get qualified teacher status (QTS) as part of an undergraduate degree course.
+### Do an undergraduate teacher training degree
 
- This could be right for you if you do not have a degree yet.
+You can get QTS as part of an undergraduate degree.
 
- You can also [get funding](/funding-your-training).
+Courses typically cost £9,250 per year and can take up to 4 years but you can [get funding](https://www.gov.uk/student-finance-calculator).
 
- <p class="call-to-action__action">
-   <a href="https://digital.ucas.com/search">Find an undergraduate <span>course</span></a>
- </p>
+Qualifications vary depending on the course. For example, you could get QTS with a:
 
-$assessment-only$
+* Bachelor of Arts (BA)
+* Bachelor of Education (BEd)
+* Bachelor of Science (BSc)
 
-If you’re an experienced teacher with a degree, you can gain QTS without doing a teacher training programme.
+<div class="training-attributes">
+  <h4>Undergraduate teacher training</h4>
+  <div class="badges">
+    <span class="badge pink">Costs £9,250 per year</span>
+    <span class="badge dark-cyan">Funding available</span>
+    <span class="badge purple">Up to 4 years</span>
+    <span class="badge dark-blue">Qualifications vary</span>
+  </div>
+</div>
 
-You’ll need to show evidence that you meet the standards for QTS.
+<a href="https://digital.ucas.com/search" class="button">Find an undergraduate course</a>
 
-A number of universities and School Centred Initial Teacher Training programmes (SCITTs) offer assessment only routes. Their criteria for entry may vary.
+## If you are already an experienced teacher
 
-You can apply to them directly.
+You may be able to get QTS through a 12-week 'assessment only' programme if you also have a degree.
 
-<p class="call-to-action__action">
-  <a href="/assessment-only-providers">See assessment only <span>providers</span></a>
-</p>  
+You’ll need to show evidence that you already meet the standards for QTS.
```

Comment:
@EmmaFrith, what do you reckon to adding another line on the 'typical' requirements here for AO, e.g. X years experience, working in 2 schools?

Reply:
yus deffo. need to pull something together still

Reply:
added a commit for this